protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -350,6 +350,9 @@ enum CoreOperatorType {
   PCAPNG_SUB_SCAN = 55;
   RUNTIME_FILTER = 56;
   ROWKEY_JOIN = 57;
+  STATISTICS_AGGREGATE = 58;
```

Comment:
Remember to also generate the C++ source for the protobuf changes (for details see protocol/readme.txt).  These will be used by ODBC driver.

Reply:
DOne